app/models/tag_user.rb

```diff
@@ -4,6 +4,18 @@ class TagUser < ActiveRecord::Base
   belongs_to :tag
   belongs_to :user
 
+  scope :notification_level_visible, -> (notification_levels) {
```

Comment:
Making this a scope as I need to reuse it on `basic_user_serializer.tags_with_notification_level`